src/IterableAction.ts

```diff
@@ -1,3 +1,25 @@
+/**
+ * Enum representing the source of IterableAction.
+ */
+export enum IterableActionSource {
+  push = 0,
+  appLink = 1,
+  inApp = 2,
+}
+
+/**
+ * Enum representing what level of logs will Android and iOS project be printing on their consoles respectively.
+ */
+export enum IterableLogLevel {
+  debug = 1,
+  info = 2,
+  error = 3,
+}
+
+/**
+ * IterableAction represents an action defined as a response to user events.
+ * It is currently used in push notification actions (open push & action buttons).
```

Comment:
This is also used for action buttons for in-app and embedded.

Reply:
We should also add a description for `IterableActionContext`

Reply:
It's done in a future PR